esphome/components/seeed_mr60fda2/button/__init__.py

```diff
@@ -0,0 +1,45 @@
+import esphome.codegen as cg
+from esphome.components import button
+import esphome.config_validation as cv
+from esphome.const import (
+    DEVICE_CLASS_RESTART,
+    DEVICE_CLASS_UPDATE,
+    ENTITY_CATEGORY_DIAGNOSTIC,
+    ENTITY_CATEGORY_NONE,
+)
+
+from .. import CONF_MR60FDA2_ID, MR60FDA2Component, mr60fda2_ns
+
+DEPENDENCIES = ["seeed_mr60fda2"]
+
+GetRadarParametersButton = mr60fda2_ns.class_("GetRadarParametersButton", button.Button)
+ResetRadarButton = mr60fda2_ns.class_("ResetRadarButton", button.Button)
+
+CONF_GET_RADAR_PARAMETERS = "get_radar_parameters"
+CONF_RESET_RADAR = "reset_radar"
+
+CONFIG_SCHEMA = {
+    cv.GenerateID(CONF_MR60FDA2_ID): cv.use_id(MR60FDA2Component),
+    cv.Optional(CONF_GET_RADAR_PARAMETERS): button.button_schema(
+        GetRadarParametersButton,
+        device_class=DEVICE_CLASS_UPDATE,
+        entity_category=ENTITY_CATEGORY_NONE,
+    ),
+    cv.Optional(CONF_RESET_RADAR): button.button_schema(
```

Comment:
I think to make this clearer, it should use `CONF_FACTORY_RESET`
```suggestion
    cv.Optional(CONF_FACTORY_RESET): button.button_schema(
```